### ISSUE TITLE
Likelihood. Did we infer it correctly in Machine Learning space.

### ISSUE THREAD

**Issue by anchalkhare** [PyDataPune/Talks#31]
In this session I will introduce audience to the concept of likelihood in Machine learning. With live code demo we will see the inference of likelihood in decision tree and how this inference can we tweaked to mold model outputs in business context . We will be using Sklearn library and pharma data.

I want to target this session to audience having intermediate python knowledge and who are eager to understand ML inference for business insights.

I expect my session to be 45 min long. I will bring my own laptop for session.

Thanks

**Comment by mayankskb** [comment]
Hi Anchal,

Thanks for making an issue.

Thanks
Mayank Mishra

**Comment by mayankskb** [comment]
Hi Anchal,

We would like to have your talk for Oct 05, 2019 @ TomTom. Can you please confirm your availability for the same.

Thanks
Mayank Mishra

**Comment by mayankskb** [comment]
Hi Anchal,

We would like to have your talk for Oct 05, 2019 @ TomTom. Can you please confirm your availability for the same.

Thanks
Mayank Mishra

**Comment by anchalkhare** [comment]
Hi Mayank,

Sure. Looking forward to it.

Regards,
Anchal

**Comment by mayankskb** [comment]
Hey,

We will be recording your session. So if you have slides/demo/paper. Kindly keep it handy.

Also, you need to install screen recording software which can capture the presentation on your laptop.

If you have any other alternate go with that if not then please go through the following readme to get an available screen recorder for your OS:

https://github.com/sigmakappa/ScreenRecorders

Let me know if a screen recorder for your OS is not available here.

Also, after installing do testing of it that it is working properly or not.

Also, please give confirmation, once done with screen recorder testing stuff.

Thanks
Mayank Mishra

**Comment by mayankskb** [comment]
Confirming your talk for Oct 05, 2019 @ TomTom

**Comment by anchalkhare** [comment]
Sure Mayank.

**Comment by mayankskb** [comment]
Hi,

Please fill this form: https://forms.gle/h784Vv3cd2D6HY4M9

Thanks 
Mayank Mishra

**Comment by mayankskb** [comment]
Hi,

Please share a brief writeup about yourself for introducing you at the meetup. Also, let me know the status of screen recording software or any intricacies if there are.

Thanks
Mayank Mishra

**Comment by anchalkhare** [comment]
Hi Mayank,

I will share the writeup by tomorrow and will let you know about
installation issues of screen recording software if any.

Regards,
Anchal

**Comment by anchalkhare** [comment]
Hi Mayank,

Here is the brief write-up about me.

Anchal Khare is having more than 8 years of data analytics and data science
industry experience. During the stint in Fractal, Infosys and ZS he has
worked in both business and operational use cases of machine learning and
deep learning. He has worked on multiple domains like Pharma/healthcare
,CPG, Finance, etc but personally healthcare domain seems to be more
challenging. He is working on patient level AI solutions in current
organization. He is a completely hands-on person and has contributed in ML
and DL projects from the delivery side and seen complete project pipeline.
Although he has gained expertise in molding AI solutions into business
context which can make business impact but he also enjoys digging deep into
nuances of ML algorithms for better understanding. For eg: when is softmax
layer used and when is sigmoid layer used in NNs

I have downloaded the software and has explored a little bit.

Regards,
Anchal

**Comment by mayankskb** [comment]
Thanks Anchal. See you on Oct 05, 2019

**Comment by anchalkhare** [comment]
Thanks Mayank. Sure.

**Comment by anchalkhare** [comment]
Hey Mayank,

How many minutes I will be given for this session?

**Comment by mayankskb** [comment]
Hi Anchal,

Usually, we try to keep talk length for 45 mins only. Let me know how much you will need to execute your talk.

Thanks
Mayank Mishra

**Comment by anchalkhare** [comment]
Hey Mayank,

45 mins seems fine to me.

Regards
Anchal

**Comment by mayankskb** [comment]
Hi Anchal,

Please Upload your screen recording to this folder. 
Link: https://drive.google.com/drive/folders/1NvszNmVAP-T5QwLH4yuBpQJWIfHudBm7?usp=sharing

Let me know if you cannot  access the folder

Thanks,
Mayank Mishra

**Comment by mayankskb** [comment]
Hey,

Please upload your video to the above-mentioned folder or do upload it to any other folder on you gdrive and then share it at pydata.pune@gmail.com

**Comment by PyDataPune** [comment]
Hi Anchal,

We didnot get your screen recording yet. Please upload it somewhere may be on your gdrive and share the link at pydata.pune@gmail.com. We will copy it from there.

for now, closing this issue, as the talk has been delivered. Feel free to make new proposals at any time.

Thanks
PyData Pune Admins
